package.json

```diff
@@ -124,10 +124,10 @@
     "rrweb-player": "^0.4.6",
     "scroll-to-element": "^2.0.0",
     "sprintf-js": "1.0.3",
-    "style-loader": "^0.23.1",
-    "svg-sprite-loader": "^3.9.0",
+    "style-loader": "^2.0.0",
+    "svg-sprite-loader": "^5.2.1",
     "svgo": "^1.0.3",
-    "svgo-loader": "^2.1.0",
+    "svgo-loader": "^3.0.0",
```

Comment:
Actually removing these in https://github.com/getsentry/sentry/pull/24582, but will wait until that's merged to rebase this.